src/babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    presets: [
+        ['@babel/preset-env', { bugfixes: true }],
+        ['@babel/preset-react', { runtime: 'automatic' }],
+        ['@babel/preset-typescript'],
+    ],
+    plugins: [
+        '@atlaskit/tokens/babel-plugin',
+        [
+            '@compiled/babel-plugin',
```

Comment:
This should only be enabled in test environments, or it will conflict with the webpack loader. What's the main intention of creating a new babel config?